runtime-kernel/linux-kernel-rpi64-lts/autobuild/postinst.in

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+LOCAL_VER_RPI4=@LOCAL_VER_RPI4@
+LOCAL_VER_RPI5=@LOCAL_VER_RPI5@
+VER=@VER@
+KERNEL_RPI4=${VER}${LOCAL_VER_RPI4}
+KERNEL_RPI5=${VER}${LOCAL_VER_RPI5}
+
+if [ -x /usr/bin/depmod ]; then
+  depmod ${KERNEL_RPI4}
+  depmod ${KERNEL_RPI5}
+else
+  echo "Kmod is not installed, please install package 'kmod'."
+fi
+
+if [ -x /usr/bin/dkms ]; then
+  dkms autoinstall --kernelver ${KERNEL_RPI4}
+  dkms autoinstall --kernelver ${KERNEL_RPI5}
+fi
+
+if [ -x /etc/kernel/postinst.d/apt-auto-removal ]; then
+  /etc/kernel/postinst.d/apt-auto-removal
+fi
+
+if [ -d "/tree/runtime-kernel" ]; then
```

Comment:
```suggestion
if ! grep -q /boot/rpi /proc/mounts ; then
```


You can using the following command to detect if `/boot/rpi` is mounted:

```
grep -q /boot/rpi /proc/mounts
```

And you can use `systemd-detect-virt` to detect if we are running in a container (not chroot):
```
systemd-detect-virt -c
```